src/PIL/Image.py

```diff
@@ -3774,7 +3802,7 @@ def _get_ifd_dict(self, offset, group=None):
             return self._fixup_dict(info)
 
     def _get_head(self):
-        version = b"\x2B" if self.bigtiff else b"\x2A"
+        version = b"\x2b" if self.bigtiff else b"\x2a"
```

Comment:
This is minor, but what is the reason behind this change? Do you just think lowercase is better, or is it more consistent with other parts of Pillow?

Reply:
I though pre-commit slipped that change in. Seems not, so it's probably something my IDE did. I'll revert this unnecessary change.